tools/data_converter/create_gt_database.py

```diff
@@ -175,8 +175,10 @@ def create_groundtruth_database(dataset_class_name,
             dict(type='LoadPointsFromFile', load_dim=5, use_dim=5),
             dict(
                 type='LoadPointsFromMultiSweeps',
-                sweeps_num=10,
-            ),
+                sweeps_num=9,
```

Comment:
We may still use `sweeps_num=10` because this best optimize the training settings.